legend-engine-xts-persistence/legend-engine-xt-persistence-protocol/src/main/java/org/finos/legend/engine/protocol/pure/v1/model/packageableElement/persistence/PersistenceContext.java

```diff
@@ -14,19 +14,16 @@
 
 package org.finos.legend.engine.protocol.pure.v1.model.packageableElement.persistence;
 
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import java.util.List;
 import org.finos.legend.engine.protocol.pure.v1.model.context.PackageableElementPointer;
 import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.PackageableElement;
 import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.PackageableElementVisitor;
 import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.connection.Connection;
 import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.persistence.context.PersistencePlatform;
 import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.persistence.service.parameter.ServiceParameter;
 
-import java.util.List;
-
 public class PersistenceContext extends PackageableElement
 {
-    @JsonSerialize(converter = PackageableElementPointer.ToPathSerializerConverter.class)
```

Comment:
Check if studio can handle it.

Reply:
needs studio test - https://github.com/finos/legend-studio/pull/3713/